### ISSUE TITLE
Support Marathon 1.5 networking API

### ISSUE THREAD

**Issue by janisz** [allegro/marathon-consul#230]
Marathon 1.5 will introduce new Networks API for apps. More information could be found:
* [Proposal](https://docs.google.com/document/d/1jTnuHLu6TVJ9FSLG7_L-ZMsUNlO-edSCwlHu3OVgANs/edit#)
* [Commit: 1fa0d9d1d](https://github.com/mesosphere/marathon/commit/1fa0d9d1d0e36e71f70efd83bd358fa8c10ecd9b)
* [Documentation PR](https://github.com/mesosphere/marathon/pull/5391)

**Comment by zanes2016** [comment]
+1 this prevents us from using marathon-consul with Marathon 1.5+

**Comment by janisz** [comment]
Oh,  we need to fix it shortly. The changes will not be big but we need to test it.

@zanes2016 Thanks for reminding me to work on this. Would you like to contribute a patch? I can give you any help you need.

**Comment by zanes2016** [comment]
@janisz I won't have time in the next few weeks. Maybe someone else can take it up

**Comment by zanes2016** [comment]
@janisz I'm looking into submitting a patch for this; however I'm running into issues building the binary from source:
```
$ make build
...
go test -coverprofile=coverage/marathon-consul.coverprofile -race _/Users/<user>/dev0/marathon-consul
can't load package: package _/Users/<user>/dev0/marathon-consul: cannot find package "_/Users/<user>/dev0/marathon-consul" in any of:
	/Users/<user>/dev0/marathon-consul/src/_/Users/<user>/dev0/marathon-consul (from $GOROOT)
	/Users/<user>/go/src/_/Users/<user>/dev0/marathon-consul (from $GOPATH)
make: *** [_/<path>/marathon-consul] Error 1
$
```
Any suggestions? This occurs on both OSX and Ubuntu 16.04.

**Comment by janisz** [comment]
What Go version are you using? It looks like you do not have configured `GOHOME` nor `GOROOT` properly.

**Comment by zanes2016** [comment]
You're correct, i don't have GOHOME or GOROOT set; but as long as GOPATH is set, it should work, no?
```
$ go version
go version go1.9.1 darwin/amd64
$ echo $GOPATH
/Users/dev/dev0/marathon-consul
$ echo $GOROOT

$ echo $GOHOME

```

**Comment by zanes2016** [comment]
Aw... However, I do see it in the $GOBIN path:
```
$ echo $GOBIN/marathon-consul
/Users/dev/dev0/marathon-consul/bin/marathon-consul
$
```

**Comment by pbetkier** [comment]
Released as of `1.5.0`.
